### PR TITLE
Add a new FMN rule to get notification about a project on pagure based on its tags

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

**Comment by pypingou** [comment]
Only tested in a shell, not on actual messages

File: fmn/rules/pagure.py [hunk 1]
```diff
@@ -41,6 +41,33 @@ def pagure_specific_project_filter(config, message, project=None, *args, **kw):
     return valid
 
 
+@hint(categories=['pagure'], invertible=False)
+def pagure_specific_project_tag_filter(config, message, tags=None, *args, **kw):
+    """ Particular pagure tag projects
```
Comment (ralphbean) [inline fmn/rules/pagure.py]: Maybe change this first line of the docstring to "Particular pagure project tags"

**Comment by ralphbean** [comment]
Only one cosmetic change needed.  The code is very clean!  :+1:

**Comment by pypingou** [comment]
Adjust and thanks for the compliment :)

**Comment by ralphbean** [comment]
L:eyes:ks good to me :p

**Pull request merged**